packages/swarm-docs/src/components/propstable.jsx

```diff
@@ -5,7 +5,7 @@ const PropsTable = props => {
   const { propMetaData = { props: {} },...rest } = props;
 
   return (
-    <table {...rest}>
+    <table {...rest} className="props-table">
```

Comment:
any concern about this preventing a parent from supplying its own `className`? Do we need any sort of className merging logic here? Totally fine as-is if we don't need to support custom classes at this level

Reply:
I don't think we need to support custom classes on the propstable. As of now, I don't see a reason for the style to be conditional.